libraries/AP_Mount/AP_Mount.h

```diff
@@ -106,6 +106,10 @@ class AP_Mount
     enum MAV_MOUNT_MODE get_mode() const { return get_mode(_primary); }
     enum MAV_MOUNT_MODE get_mode(uint8_t instance) const;
 
+    // get_rc_invalid_mode - returns rc invalid pointing coordinates of mount (i.e. no change, Retracted, Neutral)
```

Comment:
appears you did everything! but still need to get rid of this, right?

Reply:
Done!